source/ranges.tex

```diff
@@ -3803,6 +3813,97 @@
 Initializes \exposid{r_} with \tcode{std::move(t)}.
 \end{itemdescr}
 
+\rSec2[range.as.rvalue]{As rvalue view}
+
+\rSec3[range.as.rvalue.overview]{Overview}
+
+\pnum
+\tcode{as_rvalue_view} presents a \libconcept{view} of an underlying sequence
+with the same behavior as the underlying sequence
+except that its elements are rvalues.
+Some generic algorithms can be called with a \tcode{as_rvalue_view}
+to replace copying with moving.
+
+\pnum
+\indexlibrarymember{as_rvalue}{views}%
+The name \tcode{views::as_rvalue} denotes
+a range adaptor object\iref{range.adaptor.object}.
+Let \tcode{E} be an expression and let \tcode{T} be \tcode{decltype((E))}.
+The expression \tcode{views::as_rvalue(E)} is expression-equivalent to:
+\begin{itemize}
+\item
+\tcode{views::all(E)} if \tcode{same_as<range_rvalue_reference_t<T>, range_reference_t<T>>} is \tcode{true}.
+\item
+Otherwise, \tcode{ranges::as_rvalue_view(E)}.
```

Comment:
```suggestion
Otherwise, \tcode{as_rvalue_view(E)}.
```
No CPO in `views::` explicitly qualifies its corresponding template with `ranges::` when describing what it's expression-equivalent to.